activejob/lib/active_job/max_retry.rb

```diff
@@ -0,0 +1,48 @@
+module ActiveJob
+  module MaxRetry
+    extend ActiveSupport::Concern
+
+    # Includes the ability to override the default max_attempts.
+    module ClassMethods
+
+      mattr_accessor(:default_max_retry) { nil }
+
+      # Specifies the max attempts for the job.
+      #
+      #   class PublishToFeedJob < ActiveJob::Base
+      #     max_retry 2
+      #
+      #     def perform(post)
+      #       post.to_feed!
+      #     end
+      #   end
+      def max_retry(max_retries=nil, &block)
```

Comment:
We can be consistent and use `max_retry` everywhere instead of mixing `max_attempts` and `max_retry`